.github/workflows/ci.yaml

```diff
@@ -0,0 +1,16 @@
+name: CI
+run-name: CI for ${{ github.ref }}
+
+on:
+  push:
```

Comment:
What's the use of `on push` here if the jobs only works for pull_request events?

Reply:
Cause this will be merged with the work being done in #120 . It should all be part of the same workflow